java/change_atmosphere_effect/README.md

```diff
@@ -0,0 +1,29 @@
+# Change Atmosphere Effect
+Change the appearance of the atmosphere in a scene.
```

Comment:
maybe it should be "...in a scene view." since the change atmosphere property is on the `SceneView` instead of the `ArcGISScene`